apps/system/utils/utils_pmsleep.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <arch/chip/pm.h>
+
+#ifdef CONFIG_PM
```

Comment:
Duplicated. This is already checked in Makefile.

Reply:
ok